test/src/test/java/com/distkv/dst/dmeta/client/TestServer.java

```diff
@@ -5,12 +5,14 @@
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
+import java.util.concurrent.TimeUnit;
+
 public class TestServer {
 
   @Test
   public void testPutAndGet() throws InterruptedException {
     DmetaTestUtil.startAllDmetaProcess();
-    Thread.sleep(10000);
+    TimeUnit.SECONDS.sleep(5);
```

Comment:
Why change this?

Reply:
Because it doesn't matter if it's ten or five seconds. And it's more elegant and quick